src/components/questions/Question.js

```diff
@@ -1,19 +1,85 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Segment } from 'semantic-ui-react'
+import { Grid, Segment } from 'semantic-ui-react'
 
 import withCSS from '../../lib/withCSS'
 
-const Question = ({ head, id, title }) =>
-  (<Segment>
+const Question = ({ head, id, tagList, title, type, version, lastUsed }) =>
+  (<Grid stackable className="questions">
     {head}
-    <strong>{id}</strong> - {title}
-  </Segment>)
+
+    <Grid.Row className="titleRow">
+      <Grid.Column className="title" floated="left" width="11">
+        <strong>#{id}</strong> - {title} {version > 1 && `(v${version})`}
+      </Grid.Column>
+      <Grid.Column className="box type" floated="right"><b>{type}</b></Grid.Column>
+      {
+        /* TODO vertical text align */
+        tagList.map(tag => <Grid.Column className="box tag" floated="right" key={tag}>{tag}</Grid.Column>)
+      }
+    </Grid.Row>
+    <Segment divided as={Grid.Row} className="lowerSection">
+      <Grid.Column width="13">Test</Grid.Column>
+      <Grid.Column width="3">
+        <div className="lastUsedTitle">Zuletzt verwendet</div>
+        {
+          lastUsed.map(date => <div key={date}>{date}</div>)
+        }
+      </Grid.Column>
+    </Segment>
+
+    <style jsx global>{`
```

Comment:
This way of using css is totally against the principle that styled-jsx would enable. We will need to find another solution for css-in-js.